tools/nni_cmd/nnictl.py

```diff
@@ -159,6 +159,8 @@ def parse_args():
     parser_load_experiment.add_argument('--codeDir', '-c', required=True, help='the path of codeDir for loaded experiment, \
                                    this path will also put the code in the loaded experiment package')
     parser_load_experiment.add_argument('--logDir', '-l', required=False, help='the path of logDir for loaded experiment')
+    parser_load_experiment.add_argument('--searchSpacePath', '-s', required=True, help='the path of search space file for \
```

Comment:
Some experiments do not have search space, this should be an optional parameter.

Reply:
updated.